pages/landing/patcher/_features.html

```diff
@@ -3,8 +3,8 @@ <h2>Patcher Features</h2>
   {% for feature in page.features %}
     <div class="col-xs-6 col-md-4" style="margin-bottom:20px;">
       <div class="row" style="display: flex; flex-wrap: wrap;">
-        <div class="col-xs-12 col-sm-4 hidden-print" style="padding-right: 0; padding-left: 0;">
-          <img src="{{ feature.icon }}" alt="{{ feature.title }}" style="max-width: 105px;" />
+        <div class="col-xs-12 col-sm-3 hidden-print" style="padding-right: 5px;">
```

Comment:
```suggestion
        <div class="col-xs-12 col-sm-3 hidden-print" style="padding-right: 5px; text-align: right">
```

If you add this to each icon, the spacing issues will be fixed